sys/include/uuid.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_uuid
+ * @ingroup     sys
+ * @brief       Generate system wide unique IDs
```

Comment:
With that name it maybe woth pointing out, that these "U"UIDs are not compatible to [RFC 4122].

[RFC 4122]: https://tools.ietf.org/html/rfc4122